documentation/binding-data/tutorial-flow-components-binder-beans.asciidoc

```diff
@@ -38,10 +32,28 @@ binder.forField(yearOfBirthField)
 ----
 
 [NOTE]
-Code using strings to identify properties will cause exceptions during runtime if the string contains a typo or if the name of the setter and getter methods have been changed without also updating the string.
+Be cautious when using strings to identify properties. A typo in the string, or a subsequent changes to the setter and getter method names, will result in a runtime exception.
+
+=== Using JSR 303 Bean Validation
+
+You can use `BeanValidationBinder` if you prefer to use JSR 303 Bean Validation annotations such as `Max`, `Min`, `Size`, etc.
+
+`BeanValidationBinder` extends `Binder`, and therefore has the same API, but its implementation automatically adds a bean validator which takes care of all JSR 303 constraints.
+
+To use Bean Validation annotations, you need a JSR 303 implementation like Hibernate Validator available in your classpath. You can use the following dependency in Maven:
```

Comment:
We could add here a not that the validator is often already available.

```suggestion
To use Bean Validation annotations, you need a JSR 303 implementation like Hibernate Validator available in your classpath. If your environment, like Java EE container, don't provide the implementation, you can use the following dependency in Maven:
```

Reply:
I've updated the text.